notebooks/203-meter-reader/README.md

```diff
@@ -2,16 +2,32 @@
 
 [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/openvinotoolkit/openvino_notebooks/HEAD?labpath=notebooks%2F203-meter-reader%2F203-meter-reader.ipynb)
 
-![meter](https://user-images.githubusercontent.com/91237924/166135627-194405b0-6c25-4fd8-9ad1-83fb3a00a081.jpg)
+<html>
```

Comment:
could you help to fix the layout of these pictures, it looks twisted.
<img width="675" alt="image" src="https://github.com/openvinotoolkit/openvino_notebooks/assets/91237924/eec55359-cd88-44f1-8f87-53e8b50a3e0d">